### PR TITLE
Add CORS proxy server

### LINKED ISSUE #22: Add a cors proxy server that can be run locally?
I was thinking that some people want to run their own cors proxy servers for privacy reasons, so it might be easier for them if this proj includes one, and has an npm script to start it? 

Then they could just do:
image

Something like
```js
/*
* from https://www.npmjs.com/package/cors-anywhere#example
*/

// Listen on a specific host via the HOST environment variable
const host = process.env.HOST || "0.0.0.0";
// Listen on a specific port via the PORT environment variable
const port = process.env.PORT || 8080;

const cors_proxy = require("cors-anywhere");
cors_proxy
  .createServer({
    originWhitelist: [], // Allow all origins
    requireHeader: ["origin", "x-requested-with"],
    removeHeaders: ["cookie", "cookie2"],
  })
  .listen(port, host, function () {
    console.log("Running CORS Anywhere on " + host + ":" + port);
  });
```
I can make a PR if this is something you feel is beneficial!

### PR DESCRIPTION
**Pull request by cktang88**
Fixes #22 

@belaczek would love a review if you have some time!

### CODE REVIEW

File: cors-server.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const host = process.env.HOST || "0.0.0.0";
+// Listen on a specific port via the PORT environment constiable
+const port = process.env.PORT || 8080;
+
+const cors_proxy = require("cors-anywhere");
+cors_proxy
+  .createServer({
+    originWhitelist: [], // Allow all origins
+    requireHeader: ["origin", "x-requested-with"],
+    removeHeaders: ["cookie", "cookie2"],
```
Comment (belaczek) [inline cors-server.js:15]: I think this config param is not necessary

File: README.md [hunk 2]
```diff
@@ -32,6 +32,9 @@ yarn build
 
 # 3. run built application
 yarn start
+
+# 4. (optional): Starts local CORS proxy server
+yarn cors-server
```
Comment (belaczek) [inline README.md:37]: 👍

**Review by belaczek — APPROVED** [review]
Please remove (or comment out) the removeHeaders config param. Otherwise, it is good to go.

File: cors-server.js (hunk 1 above)
Comment (cktang88) [reply cors-server.js]: good point, originally was straight from the example, removed!

**Review by belaczek — APPROVED** [review]

**Pull request merged**